graphql_compiler/schema_transformation/rename_query.py

```diff
@@ -24,21 +24,20 @@
 def rename_query(
     ast: DocumentNode, renamed_schema_descriptor: RenamedSchemaDescriptor
 ) -> DocumentNode:
-    """Translate names of types using reverse_name_map of the input RenamedSchemaDescriptor.
+    """Translate names of types/ fields using reverse_name_map of the input RenamedSchemaDescriptor.
 
-    The direction in which types and fields are renamed is opposite of the process that
-    produced the renamed schema descriptor. If a type X was renamed to Y in the schema, then
-    any occurrences of type Y in the input query ast will be renamed to X.
+    The direction in which types/ fields are renamed is opposite of the process that
+    produced the renamed schema descriptor. If a type/ field X was renamed to Y in the schema, then
```

Comment:
Is `type/ field` the proper way to use spacing around `/`? I'm no expert on orthography but this looks strange to me.